db/re/item_db.txt

```diff
@@ -3850,15 +3850,15 @@
-5737,Cactus_Hat,Cactus Hat,4,20,,300,,1,,0,0xFFFFFFFF,63,2,256,,1,0,615,{},{},{}
+5737,Cactus_Hat,Cactus Hat,4,20,,300,,1,,0,0xFFFFFFFF,63,2,256,,1,0,615,{ bonus3 bAddMonsterDropItem,952,RC_Plant,500; },{},{}
 5738,Snowman_Hat,Snowman Hat,4,20,,300,,4,,1,0xFFFFFFFF,63,2,256,,1,1,616,{ bonus bStr,1; bonus bInt,1; bonus bMdef,3; bonus2 bSubEle,Ele_Water,7; bonus2 bAddMonsterDropItem,530,100; bonus2 bAddMonsterDropItem,12354,300; bonus5 bAutoSpellwhenhit,"BA_FROSTJOKER",(getrefine()>7?5:1),20,BF_WEAPON|BF_MAGIC,0; },{},{}
-5739,Sagittarius_Crown,Sagittarius Crown,4,20,,300,,3,,1,0xFFFFFFFF,63,2,256,,1,1,617,{ bonus bDex,2; },{},{}
-5740,Sagittarius_Diadem,Sagittarius Diadem,4,20,,300,,3,,1,0xFFFFFFFF,63,2,256,,1,1,618,{ bonus bDex,2; bonus bMdef,3; },{},{}
-5741,Im_Egg_Shell_Hat,Eternal Egg Shell,4,20,,0,,1,,0,0xFFFFFFFF,63,2,256,,10,1,101,{},{},{}
-5742,Rudolf_Santa_Hat,Rudolph Santa Hat,4,20,,400,,5,,0,0xFFFFFFFF,63,2,256,,10,1,619,{ bonus bLuk,1; },{},{}
+5739,Sagittarius_Crown,Sagittarius Crown,4,20,,300,,3,,1,0xFFFFFFFF,63,2,256,,1,1,617,{ bonus bDex,2; bonus2 bSubEle,Ele_Fire,7; if (getrefine()>6) { bonus bAspdRate,2; } if (getrefine()>7) { bonus bAgi,2; } if (getrefine()>8) { bonus bLongAtkRate,5; } if (getrefine()>9) { autobonus "{ bonus bAgi,10; bonus bDex,10; }",3,10000,0,"{ specialeffect2 EF_ENHANCE; }";  } },{},{}
+5740,Sagittarius_Diadem,Sagittarius Diadem,4,20,,300,,3,,1,0xFFFFFFFF,63,2,256,,1,1,618,{ bonus bDex,2; bonus bMdef,3; bonus2 bSubEle,Ele_Fire,7; if (getrefine()>6) { bonus bVariableCastrate,-3; } if (getrefine()>7) { bonus5 bAutoSpell,"WZ_SIGHTRASHER",10,5,BF_MAGIC,0; } if (getrefine()>8) { bonus bVariableCastrate,-2; } if (getrefine()>9) { bonus bMatkRate,40; } },{},{}
+5741,Im_Egg_Shell_Hat,Eternal Egg Shell,4,20,,0,,1,,0,0xFFFFFFFF,63,2,256,,10,1,101,{ bonus2 bAddItemGroupHealRate,IG_Potion,10; bonus bHealPower2,10; },{},{}
+5742,Rudolf_Santa_Hat,Rudolph Santa Hat,4,20,,400,,5,,0,0xFFFFFFFF,63,2,256,,10,1,619,{ bonus2 bAddMonsterDropItem,538,100; bonus2 bAddMonsterDropItem,539,100; bonus2 bAddItemHealRate,538,100; bonus2 bAddItemHealRate,539,100; },{},{}
 5743,Orange_Stem_Hat,Orange Stem Hat,4,20,,100,,0,,0,0x00000000,63,2,256,,1,0,620,{ bonus bLuk,3; },{},{}
-5744,Capricon_Crown,Capricon Crown,4,20,,300,,3,,1,0xFFFFFFFF,63,2,256,,1,1,621,{ bonus bVit,2; bonus bDex,2; },{},{}
-5745,Capricon_Diadem,Capricon Diadem,4,20,,300,,3,,0,0xFFFFFFFF,63,2,256,,1,1,622,{ bonus bDex,2; bonus bMdef,5; },{},{}
+5744,Capricon_Crown,Capricon Crown,4,20,,300,,3,,1,0xFFFFFFFF,63,2,256,,1,1,621,{ bonus bVit,2; bonus bDex,2; if (getrefine()>6) { bonus3 bAutoSpell,"TK_SEVENWIND",1,500; } },{},{}
+5745,Capricon_Diadem,Capricon Diadem,4,20,,300,,3,,0,0xFFFFFFFF,63,2,256,,1,1,622,{ bonus bDex,2; bonus bMdef,5; bonus2 bSubEle,Ele_Earth,5; if (getrefine()>6) { bonus bInt,2; } if (getrefine()>7) { bonus bVariableCastrate,-3; } if (getrefine()>8) { bonus bHealPower2,4; } if (getrefine()>9) { bonus3 bAutoSpell,"BS_HAMMERFALL",5,5; } },{},{}
```

Comment:
* I found the Aegis name is `Capricorn_Diadem`, there's typo the missing 'r'
* .@r = `getrefine()`